### PR TITLE
Style up templates with bootstrap classes

### PR DESCRIPTION
**Pull request by manuscriptmastr**
@nybblr Did some substantial styling. Thoughts? Also, still can't figure out how to make borders happen. Somewhere I must be missing a class. Thanks!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -39,3 +39,4 @@ jspm_packages
 .DS_Store
 /uploads
 /dist
+/test_photos
```
Comment (nybblr) [inline .gitignore]: A good name for this folder is "fixtures". (as in, "fixture data")

File: templates/feed.pug [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+        img.img-fluid.rounded(src=`/uploads/${val.image}`).mb-2
         br
-        = val.description
+        strong #{val.coffeeType}
         br
-        = val.coffeeType
+        = val.description
         br
-        = val.createdAt
+        small #{val.createdAt}
```
Comment (nybblr) [inline templates/feed.pug]: If you don't need interpolation, it's conventional to do `small= val.createdAt` instead.

File: templates/layout.pug [hunk 3]
```diff
@@ -6,27 +6,26 @@ html(lang="en")
...
-        a(class="nav-link active" href="/") Coffeegram
-      li(class="nav-item")
-        a(class="nav-link active" href="/coffeegrams/new") Upload
-      li(class="nav-item")
-        a(class="nav-link active" href="/about") About
-      if currentUser
+    div.container.px-0.mb-4.mt-4
+      ul.nav.d-flex.flex-row.justify-content-left
+        li(class="nav-item nav-pills")
+          a(class="nav-link active" href="/") Coffeegram
```
Comment (nybblr) [inline templates/layout.pug]: Wanna go ahead and convert these to `li.nav-item.nav-pills` etc?

**Comment by nybblr** [comment]
Upload some screenies for me!

**Comment by manuscriptmastr** [comment]
screen shot 2017-03-31 at 7 54 10 pm
screen shot 2017-03-31 at 7 53 58 pm

**Comment by nybblr** [comment]
Josh, this looks GREAT! :shipit:

**Comment by manuscriptmastr** [comment]
🕺

**Pull request merged**